src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/CommentTabPanel.kt

```diff
@@ -20,11 +20,11 @@ class CommentTabPanel(
 
     fun waitForActive(): CommentTabPanel {
         driver.wait(
-            Duration.ofSeconds(45),
+            Duration.ofSeconds(120),
```

Comment:
I think Duration.ofMinutes(2) would be slightly more clear. Also if we are using this two minutes duration in most of the places I think it would be good to extract it to constant.

Reply:
I still think it is a good practice and while I get your point, I see this case as slightly different from what you put in the example. We switched from a specific durations to the default one, which is the same for all of the usages. However you are right that the memory/performance impact may be insignificant and if you think that it will be better to leave it the way it was, to stress out the we should approach each of the timeouts present here separately, I'm fine with that.

Reply:
Yes, it's good practice to deduplicate, however I claim that in this case there was no duplication, because timeouts were different for most of the operations. Even if accidentally some of them matched it wasn't because we treated it as the same value.

We could create a common default and name it e.g. `usersPatience`. This would allow us to document rationale behind the value, because as @dagguh pointed out there was research around it. However I don't think that in our case it has enough value to balance the potential risk. Deduplicating it would suggest to ourselves and future contributors that the timeouts are meant to be manipulated all at once and not with proper granularity.

Reply:
Yeah, gotta watch out for accidental duplication. Maybe some SSH timeout is also set to `Duration.ofMinutes(2)`, but it's semantically different, they shouldn't reuse the same symbol.
And don't worry about `Duration` constructor performance, it's instant (pun intended :wink:).

Reply:
I have deduplicated duration values for both new actions, as every single step on those views seems to be very slow, but I didn't touch any other timed conditions.